pkg/collections.go

```diff
@@ -25,16 +25,20 @@ package pkg
 import (
 	"context"
 	"errors"
+	"fmt"
 	"sort"
 
+	"github.com/vbauerster/mpb/v5"
+	"github.com/vbauerster/mpb/v5/decor"
+
 	"github.com/arangodb/go-driver"
 	"github.com/rs/zerolog/log"
 	"golang.org/x/sync/errgroup"
 	"golang.org/x/sync/semaphore"
 )
 
 // copyCollections copies all collections for a database.
-func (c *copier) copyCollections(ctx context.Context, db driver.Database) error {
+func (c *copier) copyCollections(ctx context.Context, db driver.Database, doneCollections chan string) error {
```

Comment:
Prefer to strong type the channel.
Something like 

type databaseAndCollection struct {
DatabaseName string 
CollectionName string
}

doneCollection chan databaseAndCollection

Reply:
I was actually leaning that way. I'll do that!